example/src/main/java/com/iopipe/examples/ExampleExecution.java

```diff
@@ -0,0 +1,54 @@
+package com.iopipe.examples;
+
+import com.iopipe.IOpipeExecution;
+import com.iopipe.plugin.IOpipePluginExecution;
+
+/**
+ * This is an execution for the plugin.
```

Comment:
Where/why would someone create an execution? (since this is in the examples dir)

Reply:
I clarified the use of this class.